solr/core/src/java/org/apache/solr/search/ReRankQParserPlugin.java

```diff
@@ -72,25 +74,33 @@ public Query parse() throws SyntaxError {
 
       double reRankWeight = localParams.getDouble(RERANK_WEIGHT, RERANK_WEIGHT_DEFAULT);
 
-      return new ReRankQuery(reRankQuery, reRankDocs, reRankWeight);
+      String reRankOperator = localParams.get(RERANK_OPERATOR, "add");
+
+      return new ReRankQuery(reRankQuery, reRankDocs, reRankWeight, reRankOperator);
     }
   }
 
   private static final class ReRankQueryRescorer extends QueryRescorer {
 
     final double reRankWeight;
+    final String reRankOperator;
```

Comment:
minor: wondering about use of `String` vs. enum (which would also take care of handling of typos e.g. `reRankOperator=multi` would throw an exception rather than be silently treated as `add`)

Reply:
Good idea, yes - I'll look into that.

Reply:
enums will also help later to introduce other ops like min, max, etc

Reply:
Looks like `.valueOf` is the magic word e.g. like in https://github.com/apache/solr/blob/releases/solr/9.1.1/solr/core/src/java/org/apache/solr/cloud/overseer/OverseerAction.java say.